Задания с 05.10.2022/Task1/Task1/FTPClient.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace SimpleFTP;
+
+using System.Net;
+using System.Net.Sockets;
+
+/// <summary>
+/// FTP-Client realization
+/// </summary>
+public class FTPClient
+{
+    private IPAddress ip;
+    private int port;
+
+    public FTPClient(string host, int port)
+    {
+        ip = IPAddress.Parse(host);
+        this.port = port;
+    }
+
+    /// <summary>
+    /// Requests file downloading from the server
+    /// </summary>
+    public async Task<string> Get(string path)
+    {
+        using (var client = new TcpClient())
```

Comment:
```suggestion
        using var client = new TcpClient();
```
И дальше без лишних отступов